src/main/java/com/shapesecurity/shift/es2017/codegen/TokenStream.java

```diff
@@ -81,7 +81,7 @@ public void put(@Nonnull String tokenStr) {
             this.lastCodePoint = lastChar;
         }
         if ((lastCodePoint == '+' || lastCodePoint == '-') && lastCodePoint == rightCodePoint ||
-                Utils.isIdentifierPart(lastCodePoint) && Utils.isIdentifierPart(rightCodePoint) ||
+                Utils.isIdentifierPart(lastCodePoint) && (Utils.isIdentifierPart(rightCodePoint) || rightCodePoint == '\\') ||
```

Comment:
I don't understand this change.

Reply:
This check is necessary to avoid outputting `let \u0073` as `let\u0073`. The default codegen will never output `\u0073`, which is why that check isn't there already, but it needs to be there to be usable with other codegens like the WebSafeCodeGen.

The WebSafeTokenStream had this check.